### DISCUSSION TITLE
BRK kwaliteitslabel kolom toevoegen?

### DISCUSSION THREAD

**Discussion by jhpoosthoek** [nlextract/NLExtract#390] [Q&A] [answered]
Voor de BRK NLExtract, zijn er nog plannen om het kwaliteitslabel toe te voegen? Zie https://c.spotler.com/ct/m2/k1/yo65einN8sZHSJH9_5Vu7-wX9wKJE4UZRCFRNs3zNT5_Yv09cT6SITck8qSYr9JfMXm2pp7EPY4jga9PowD5Cg/ZUUyeLMYfS8ir3i

**Comment by justb4** [comment, marked as answer]
Is tijdje terug al gedaan: via issue #366 .

**Reply by jhpoosthoek** [reply]
Dank!

**Comment by jhpoosthoek** [comment]
De kolom kwaliteit zit niet in mijn resultaat tabel in PSQL. Ik heb deze aangemaakt met de actuele NLExtract gedownload via git. Ik zie deze wel in NLExtract/brk/etl/gfs/brk.gfs staan. Weten jullie of er ergens in de code nog naar brk-v4.gfs wordt gerefereerd?

**Reply by justb4** [reply]
"Kwaliteit" is een aparte tabel, niet een kolom, al doet bericht Kadaster dat suggereren. Zo wordt het ook aangeleverd, in GML. De GFS beschrijft dat ook. En geldt ook per Grens niet per Perceel (BRK is uiteindelijk verzameling grenzen, Percelen worden daaruit gegenereerd). Via een ALTER TABLE ADD COLUMN+UPDATE kun je wel zelf 'kwaliteit' aan Grens tabel toevoegen.

**Reply by jhpoosthoek** [reply]
Ah kijk, dat verklaard de boel. Goed om te weten dat het per grens is!
